java/base-client/src/test/resources/token-response2.json

```diff
@@ -1,5 +1,5 @@
 {
-  "someOtherField": "abc",
-  "expiresOn": 123,
-  "accessToken": "a-token"
-}
\ No newline at end of file
+  "some_other_field": "abc",
+  "expires_on": 123, "some_field": "bca",
```

Comment:
I'm not sure how/if we should even try to handle if expiry_in and expiry_on are somehow contradicting each other. I assume not

Reply:
See other comment. There's no `expiry_on` in the OAuth2 spec.